core/src/test/scala/aima/core/environment/vacuum/ModelBasedReflexVacuumAgentSpec.scala

```diff
@@ -57,7 +57,7 @@ class ModelBasedReflexVacuumAgentSpec extends Specification {
   }
 
   "should do nothing if low on power" in new context {
-    val resultStream = Stream.continually(agent.agentFunction.apply(DirtyPercept))
+    val resultStream = LazyList.continually(agent.agentFunction.apply(DirtyPercept))
```

Comment:
Why did you change this to a list.  I see that [Stream.continually still exists in scala 2.13](https://www.scala-lang.org/api/current/scala/collection/immutable/Stream$.html#continually[A](elem:=%3EA):scala.collection.immutable.Stream[A])

Reply:
Nevermind, I see it is deprecated now `(Since version 2.13.0) Use LazyList (which is fully lazy) instead of Stream (which has a lazy tail only)` . That is too bad I don't really care for the new name.